test/test_interface.jl

```diff
@@ -310,6 +310,18 @@ end
     @test layout_getindex(T,1:10,1:10) isa BandedMatrix
 end
 
+@testset "OneElement" begin
+    o = OneElement(1, 3, 5)
+    @test bandwidths(o) == (2,-2)
+    n,m = rand(1:10,2)
+    o = OneElement(1, (rand(1:n),rand(1:m)), (n, m))
+    @test bandwidths(o) == bandwidths(BandedMatrix(o))
```

Comment:
This test is a tautology since `BandedMatrix` calls `bandwidths(o)` to determine its bandwidths. I think the following will be better:
```suggestion
  @test bandwidths(o) == bandwidths(sparse(o))
```
You'll need to add SparseArrays to the using statement